lsp/app/aws-lsp-cloudformation-binary/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "UMD",
+        "target": "ES2021",
+        "moduleResolution": "node",
+        "lib": ["ES2021"],
+        "rootDir": "./src",
+        "outDir": "./out",
+        "declaration": true,
+        "sourceMap": true,
+        "skipLibCheck": true,
+        "forceConsistentCasingInFileNames": true,
```

Comment:
the more projects we have, the more of this kind of configuration we have to paste and maintain in many places